service/src/main/java/org/sunbird/service/user/ExtendedUserProfileService.java

```diff
@@ -0,0 +1,11 @@
+package org.sunbird.service.user;
+
+import org.sunbird.request.Request;
+
+public interface ExtendedUserProfileService {
+    /**
+     * Validate json payload of user profile of a given user request
+     * @param userRequest
+     */
+    public void validateProfile(Request userRequest);
```

Comment:
Make sure this Request object contains RequestContext object for debugabilty

Reply:
Yes. And using the requestContext in the loggers